analyzers/tests/SonarAnalyzer.UnitTest/TestCases/MethodOverrideNoParams.CSharp11.cs

```diff
@@ -0,0 +1,13 @@
+﻿public interface IMath
+{
+    static virtual void Method(params int[] numbers) { }
+    static virtual void Method(string s, params int[] numbers) { }
+    static abstract void Method(string s, string s1, params int[] numbers);
+}
+
+public class Foo : IMath
+{
+    public static void Method(int[] numbers) { } // Compliant, FN
```

Comment:
This rule makes sense for implementing interface members too, however the rule only checks about overrides. I am not sure if it should be a separate rule or this one with a changed message.

Reply:
I'm in favor for extending the rule. I would let this as a FN until we take a decision.